### ISSUE TITLE
boardgame arena 熱鍵相撞

### ISSUE THREAD

**Issue by chainsawriot** [chainsawriot/hansolo#1]
大部份按鍵都會彈出 chat window

**Comment by lentychang** [comment]
`Alt` + `[a-Z]` and `Shift` + `[a-Z]` lead to type in the chat.
Maybe we could use `Ctrl` + `Shift` + `a`,`s`,`x`,`z` ?

btw, somebody is asking for extention onthe [BGA forum](https://forum.boardgamearena.com/viewtopic.php?t=13448) XD

**Comment by chainsawriot** [comment]
Actually one simple solution is to butcher the chat window so that it doesn't pop-up. No one is gonna chat when he or she is playing Solo, right?

**Comment by chainsawriot** [comment]
I've tried to butcher the chat window and the game won't progress.

I have tried this

```javascript
window.addEventListener("keydown", function(e) {
    // space and arrow keys
    if([38].indexOf(e.keyCode) > -1) {
        e.preventDefault();
        document.querySelector("#bottom_button_call").click()
    }
}, false);
```

It disables the original function of the upper arrow (38), i.e. scrolling, and makes it the Solo key. It seems to work.

I think we can then map the keys not triggering the chat window (e.g. Esc, Tab, Shift, Control, Alt, Meta) as the game's function keys.

**Comment by chainsawriot** [comment]
fixed 22ad9a70a5df8fbcc04713ef8a9b3b17087c82a4
